sources/zsh/aliases.d/jadx

```diff
@@ -0,0 +1,2 @@
+alias jadx="/opt/tools/jadx/build/jadx/bin/jadx"
+alias jadx-gui="/opt/tools/jadx/build/jadx/bin/jadx-gui"
```

Comment:
In my opinion, this shouldn't be aliases, but instead symbolic links placed in `/opt/tools/bin/`